sirepo/template/ml.py

```diff
@@ -620,11 +620,57 @@ def _generate_parameters_file(data):
             v,
             f"{d[dm.classificationAnimation.classifier]}.py",
         )
-    res += template_common.render_jinja(SIM_TYPE, v, "build-model.py")
+    res += _build_model_py(v)
     res += template_common.render_jinja(SIM_TYPE, v, "train.py")
     return res
 
 
+def _build_model_py(v):
+    def _import_layers(v):
+        i = ", ".join(n for n in v.layerImplementationNames if not n == "Dense")
+        if i:
+            return "," + i
+        return ""
+
+    def _layer_args(layer):
+        l = layer.layer
+        if l == "Activation":
+            return layer.activationActivation
+        if l == "AlphaDropout":
+            return layer.alphaDropoutRate
+        if l == "Dense":
+            return f'{layer.denseDimensionality}, activation="{layer.denseActivation}"'
+        if l == "Dropout":
+            return layer.dropoutRate
+        if l == "Flatten":
+            return ""
+        if l == "GaussianDropout":
+            return layer.gaussianDropoutRate
+        if l == "GaussianNoise":
+            return layer.gaussianNoiseStddev
+        raise ValueError(f"invalid layer.layer={l}")
+
+    def _build_layers(v):
+        res = ""
+        for i, l in enumerate(v.neuralNetLayers):
```

Comment:
`for i, l in enumerate(v.neuralNetLayers):`

Reply:
You mean `for i, l in enumerate(layers):` correct? since we pass layers into `_build_layers`